packages/core/src/App/Containers/WarningScamMessageModal/warning-scam-message-checkbox-content.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+const WarningScamMessageCheckbox = ({ acknowledgeMessage }) => (
+    <div className='warning-scam-message__checkbox-container'>
```

Comment:
could you please explain why you use somewhere one hyohen and somewhere two hyphens
warning-scam-message__checkbox-container
warning-scam-message__website-icon
warning-scam-message--content__info--title

Reply:
one hyphens represents the name of the thing exp: website-icon, scam-message, while double hyphens means the content in that particular name exp: content--title, content--header, content--footer